src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1160,6 +1231,24 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         scrollBar.Value(viewTop);
     }
 
+    // Method Description:
+    // - Called continuously to smoothly scroll viewport when user is

```

Comment:
nit: don't say "smoothly"

I know what you mean, but there's actually some issues about supporting smooth scroll and that's actually a different thing.